src/components/organisms/UserProfileModal/UserProfileModal.tsx

```diff
@@ -159,6 +164,12 @@ const Badges: React.FC<{ user: WithId<User> }> = ({ user }) => {
   );
   const venues = useSelector(orderedVenuesSelector);
 
+  const { relatedVenues } = useConnectRelatedVenues({
+    venueId: currentVenue.id,
+    withEvents: false,
+  });
+  const relatedVenueIds = relatedVenues.map((venue) => venue.id);
+
   const playaTime = useMemo(() => {
```

Comment:
Is this just named poorly, or is it something playa-only that we don't want to show here?